src/r2mm/installing/default_installation_rules/InstallationRuleApplicator.ts

```diff
@@ -149,6 +149,7 @@ export default class InstallationRuleApplicator {
             buildBepInExRules("Distance"),
             buildBepInExRules("FiveNightsAtFreddysIntoThePit"),
             buildBepInExRules("GoreBox"),
+            buildBepInExRules("TCG Card Shop Simulator"),
```

Comment:
The string value used here doesn't match the one used in `InstallationRuleApplicator.ts`, preventing the app from building.

Reply:
Should be resolved now